src/azure-cli/azure/cli/command_modules/synapse/_help.py

```diff
@@ -0,0 +1,275 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.help_files import helps  # pylint: disable=unused-import
+
+# pylint: disable=line-too-long, too-many-lines
+
+
+helps['synapse'] = """
+type: group
+short-summary: Manage and operate Synapse Workspace, BigDataPool, SqlPool.
+"""
+
+helps['synapse workspace'] = """
+type: group
+short-summary: Manage Synapse Workspace.
+"""
+
+helps['synapse workspace create'] = """
+type: command
+short-summary: create a synapse workspace.
+long-summary: create a synapse workspace.
```

Comment:
long-summary is unnecessary as it's the same as short-summary. Please also remove other long-summary if they are the same as short-summary.

Reply:
> long-summary is unnecessary as it's the same as short-summary. Please also remove other long-summary if they are the same as short-summary.

Fixed